pyvex/lift/__init__.py

```diff
@@ -9,62 +17,51 @@ class Lifter(object):
     REQUIRE_DATA_C = False
     REQUIRE_DATA_PY = False
 
-    def __init__(self,
-            irsb,
-            data,
-            max_inst,
-            max_bytes,
-            bytes_offset,
-            opt_level,
-            traceflags,
-            allow_lookback):
-        self.irsb = irsb
+    def __init__(self, arch, addr):
+        self.arch = arch
+        self.addr = addr
+
+    def _lift(self,
+             data,
+             bytes_offset=None,
+             max_bytes=None,
+             max_inst=None,
+             opt_level=1,
+             traceflags=None,
+             allow_lookback=None):
+        """
+        Create an irsb from data
+
+        All of the lifters will be used in the order that they are registered
+        whenever an instruction cannot be decoded until one is able to decode it
+        """
```

Comment:
The parameters to these were always a bit mysterious.  I know some of these get defined a few other places, but seeing as this is a base class for a few other important things (and for people to write more Lifters sans-Gymrat), perhaps we should lay down a real docstring here?